pom.xml

```diff
@@ -155,4 +203,39 @@
     </profile>
   </profiles>
 
+  <build>
```

Comment:
I'll have to remove it later on. It is a problem with Eclipse. Eclipse fails loading the project when it does not know what to do with a plugin - here with the licence-mave-plugin. It attempted to install Sonar's m2e-error-disabler-plugin, but Eclipse failed (NPE...) doing so. Will try with new yearly Eclipse version, expected to come out today.